assets/css/home.css

```diff
@@ -96,7 +92,7 @@ html {
     @apply p-3 overflow-scroll;
 }
 
-.content p code, li code {
+.content p code, li code, table code {
```

Comment:
just my personal preference, gray background inside the table doesn't look good to me :-)

Reply:
yeah, it is just my personal preference and just wanted to know the other's thoughts. Anyway, I am fine, and thank you for your nice work!